DQM/SiPixelMonitorTrack/src/SiPixelTrackResidualSource.cc

```diff
@@ -894,6 +921,12 @@ void SiPixelTrackResidualSource::analyze(const edm::Event& iEvent, const edm::Ev
   //iEvent.getByLabel( clustersrc_, clusterColl );
   iEvent.getByToken( clustersrcToken_, clusterColl );
   auto const & clustColl = *(clusterColl.product());
+
+  // get digis
+  edm::Handle< edm::DetSetVector<PixelDigi> >  digiinput;
+  iEvent.getByToken( digisrcToken_, digiinput );
+  if (!digiinput.isValid()) { cout << "no digis found\n"; }
```

Comment:
cout is not allowed. please use the message logger to raise a warning.
thanks,
F.